ui/blocks/Onboarding.tsx

```diff
@@ -96,7 +96,7 @@ export const Onboarding: React.FC<OnboardingProps> = ({
               <Input name="resident" label="Bewohnername" />
             </>
           )}
-          {area.companyNeedToShowCoronaTest > 0 && (
+          {area.companyNeedToShowCoronaTest > 0 && area.testException != 'tg' && (
```

Comment:
Wow, why do we need these magic string comparisons? Also, lets use `===` and `!==` except for comparison with `null` :)